assets/apps/dashboard/src/Components/Content/PromotionsContent.js

```diff
@@ -0,0 +1,5 @@
+const promotions = {
+	bf22: require('../../../../../img/dashboard/bf22.png'),
+};
+
+export default promotions;
```

Comment:
Is there any reason for using `require` instead of importing the image here? 
```suggestion
import bf22 from '../../../../../img/dashboard/bf22.png';

export default {
	bf22,
};
```